spec/spec_helper.rb

```diff
@@ -29,7 +29,6 @@
 
 RSpec.configure do |config|
   config.before(:suite) do
-    Webservices::Application.model_classes.each(&:recreate_index)
```

Comment:
This is unrelated to zip codes sorting but also a one liner, so I put it together :-P

From the commit message:

    Stop recreating indexes at the beginning of the test suite
    
    Tests already do this for the indexes they need, this one is redundant
    and causes running a single spec a lot slower since it needs to wait for
    all unrelated indexes to be recreated.